.dotnet/api/OpenAI.netstandard2.0.cs

```diff
@@ -1207,7 +1210,9 @@ public class ChatCompletionOptions : IJsonModel<ChatCompletionOptions>, IPersist
         public IDictionary<int, int> LogitBiases { get; }
         public int? MaxOutputTokenCount { get; set; }
         public IDictionary<string, string> Metadata { get; }
+        public ChatMessageContent PredictedContent { get; set; }
         public float? PresencePenalty { get; set; }
+        public ChatReasoningEffort? ReasoningEffort { get; set; }
```

Comment:
Does this warrant a `Level` suffix or similar? On one hand: that's the name in REST. On the other: the property name isn't indicative of an enum value; I'd blindly assume it was a number of some form before I'd think of it being an enum.

Reply:
I think `ChatReasoningEffortLevel` sounds reasonable. We did this for `ChatImageDetailLevel`, so it's consistent with the precendent. 🙂